File: src/main/java/dev/vrba/slack/tenor/bot/controller/HomeController.java
package dev.vrba.slack.tenor.bot.controller;

import io.micronaut.core.annotation.NonNull;
import io.micronaut.http.HttpResponse;
import io.micronaut.http.annotation.Controller;
import io.micronaut.http.annotation.Get;

import java.net.URI;

@Controller
public class HomeController {

    @Get
    @NonNull
    public HttpResponse<Void> index() {
        return HttpResponse.redirect(URI.create("https://github.com/jirkavrba/slack-tenor-bot"));
    }

}

File: src/main/kotlin/dev/vrba/slack/controller/HomeController.kt
package dev.vrba.slack.controller

import io.micronaut.http.HttpResponse
import io.micronaut.http.annotation.Controller
import io.micronaut.http.annotation.Get
import java.net.URI

@Controller
class HomeController {
    @Get
    fun index(): HttpResponse<Unit> {
        return HttpResponse.redirect(URI.create("https://github.com/jirkavrba/slack-tenor-bot"))
    }
}
